client/landing/gutenboarding/gutenboard.tsx

```diff
@@ -56,6 +57,14 @@ export function Gutenboard() {
 	// (and would lead to weird mounting/unmounting behavior).
 	const onboardingBlock = useRef( createBlock( name, {} ) );
 
+	const {
+		location: { pathname },
+	} = useHistory();
+
+	useEffect( () => {
+		recordTracksPageViewWithPageParams( `/gutenboarding${ pathname }` );
```

Comment:
Just noticed that when I navigate to `/gutenboarding` and look at the `calypso_page_view` event in the network inspector, it's sending a trailing slash in the path i.e. `path=/gutenboarding/`. Not sure how much it matters (I don't know every way tracks data is used 😅), it might be annoying for folks designing their funnels using tracks data. But fwiw events from the `/start` onboarding flow don't have a trailing slash.

Reply:
hmm it's different but I don't think we should add code to change it unless it does turn out to cause any issues for the analytics people, the other routes like `/gutenboarding/start` don't have a trailing `/`, so the `pathname` must be `/` here to give us `/gutenboarding/`. I think it's alright as long as they can track the unique URLs